Gruntfile.js

```diff
@@ -33,6 +33,12 @@ module.exports = function( grunt ) {
 			webpack_production: {
 				command: 'cross-env BABEL_ENV=production webpack'
 			},
+			composer_production: {
+				command: 'rm -rf vendor && composer install --no-dev'
```

Comment:
This `rm -rf vendor` here is problematic because it blows away any Git repos that are installed via `composer install --prefer-source`. If this is to be done, then it should temporarily rename `vendor` to `vendor-dev` and then afterward delete the new `vendor` and rename `vendor-dev` back to its original location.